src/current/v24.3/troubleshoot-lock-contention.md

```diff
@@ -0,0 +1,395 @@
+
+When Transaction 1, releases key `k=2`, Transaction 2 should output the following:
+
+~~~
+  k |  v
+----+-------
+  2 | 2012
+(1 row)
+~~~
+
+`COMMIT` Transaction 2:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+COMMIT;
+~~~
+
+**Terminal 3**
+
+`COMMIT` Transaction 3 and verify that the `UPDATE` has succeeded:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+COMMIT;
+SELECT * FROM t where k=2;
+~~~
+
+The `SELECT` statement should output the following:
+
+~~~
+  k |  v
+----+-------
+  2 | 2032
+(1 row)
+~~~
+
+## Identify waiting and blocking transactions
+
+This section of the tutorial uses the [**Insights** page]({% link {{ page.version.version }}/ui-insights-page.md %}#transaction-executions-view) of the DB Console to identify waiting and block transactions in the demo cluster. With a CockroachDB Cloud cluster, the Cloud Console has a similar [**Insights** page]({% link cockroachcloud/insights-page.md %}#transaction-executions-view). You can also use the [`crdb_internal`]({% link {{ page.version.version }}/performance-recipes.md %}#identify-transactions-and-objects-that-experienced-lock-contention) system catalog to view tables and indexes that experienced contention.
+
+While this tutorial uses the data from [Example 1](#example-1), when troubleshooting lock contention in your own workload, you can adapt the following steps using the DB Console or the Cloud Console.
+
+### High Contention Insights
+
+After executing the three transactions in the previous section, open the [DB Console](#db-console) for the demo cluster. Navigate to the **Insights** page and select **Workload Insights** > **Transactions Executions** view.
+
+<img src="{{ 'images/v24.3/troubleshoot-lock-contention-transaction-executions-view.png' | relative_url }}" alt="Transaction Executions view" style="border:1px solid #eee;max-width:100%" />
+
+Depending on when you executed the three transactions, to display the transactions flagged with insights, you may have to select a longer time interval, such as **Past 6 Hours**.
+
+<img src="{{ 'images/v24.3/troubleshoot-lock-contention-time-interval.png' | relative_url }}" alt="Time interval" style="border:1px solid #eee;max-width:100%" />
+
+With an adequate time interval, two [**High Contention**]({% link {{ page.version.version }}/ui-insights-page.md %}#high-contention) insights will be listed for the waiting transactions that experienced contention, **Transaction 2** and **Transaction 3** executed in Example 1.
+
+<img src="{{ 'images/v24.3/troubleshoot-lock-contention-high-contention.png' | relative_url }}" alt="High Contention" style="border:1px solid #eee;max-width:100%" />
+
+### Waiting statement
+
+To identify the exact statement in the transaction that experienced high contention, click the **Latest Transaction Execution ID**
+452da3e7-e7fa-4801-aa40-c59988d14eb6, the ID of the latest execution with the given [transaction fingerprint]({% link {{ page.version.version }}/ui-transactions-page.md %}).
```

Comment:
i think the UI transactions page link should probably go to the subsection at anchor `#transaction-fingerprints-view`?